packages/webpack-config/src/index.js

```diff
@@ -1,55 +1,76 @@
+const argv = require('yargs').argv;
 const autoprefixer = require('autoprefixer')();
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 const webpack = require('webpack');
 const dd = require('./dd.js');
 
+let mode = 'production';
+if (argv.mode === 'development' || argv.mode === 'production') {
+	mode = argv.mode;
+} else if (process.env.WEBPACK_SERVE) {
+	mode = 'development';
+}
+
 dd();
-console.log('Webpack Config Deloitte');
+console.log('Deloitte Digital Webpack Config');
 console.log('Current working directory:', process.cwd());
+console.log('Mode:', mode);
```

Comment:
I like this, what do you think about only logging the directory and mode when the --verbose flag is present? Just an idea.

Reply:
Good idea. That's consistent with Webpack only showing `PublicPath: /dist/` when the `--verbose` flag is present.